core/src/main/java/de/jplag/SubmissionSetBuilder.java

```diff
@@ -31,19 +31,14 @@ public class SubmissionSetBuilder {
 
     private static final Logger logger = LoggerFactory.getLogger(SubmissionSetBuilder.class);
 
-    private final Language language;
     private final JPlagOptions options;
-    private final Set<String> excludedFileNames; // Set of file names to be excluded in comparison.
 
     /**
      * Creates a builder for submission sets.
-     * @param language is the language of the submissions.
      * @param options are the configured options.
      */
-    public SubmissionSetBuilder(Language language, JPlagOptions options) {
-        this.language = language;
+    public SubmissionSetBuilder(JPlagOptions options) {
         this.options = options;
-        this.excludedFileNames = options.excludedFiles();
```

Comment:
JPlag has to ensure backward compatibility. Therefore, you could mark the old constructor as deprecated and just pass the option to a new constructor. But we cannot remove the constructor directly.

Reply:
Is the SubmissionSetBuilder part of the interface? I though it was entirely internal, thus making backwards compatability unnecessary

Reply:
It's a public class in the core module. Therefore, I think we have to consider it as part of the interface